package.json

```diff
@@ -2,16 +2,20 @@
   "name": "poolifier",
   "version": "1.2.1",
   "description": "Library on top of node js worker threads that implement various worker pools type",
-  "main": "index.js",
-  "types": "index.d.ts",
+  "main": "lib/index.js",
   "scripts": {
-    "build": "npm install",
-    "test": "standard && nyc mocha --exit --timeout 20000 tests/*test.js ",
-    "debug-test": "mocha --inspect-brk --exit tests/*test.js ",
-    "demontest": "nodemon --exec \"npm test\"",
+    "build": "npm run build:clean && tsc",
+    "build:clean": "rimraf lib",
+    "test": "standard && ts-standard && npm run build && nyc mocha --exit --timeout 20000 tests/*test.js ",
+    "test:debug": "mocha --inspect-brk --exit tests/*test.js ",
+    "test:nodemon": "nodemon --exec \"npm test\"",
     "coverage": "nyc report --reporter=text-lcov | coveralls",
-    "standard-verbose": "npx standard --verbose",
-    "lint": "prettier-standard --format | standard --fix"
+    "import-sort": "npx import-sort-cli --write 'src/**/*.ts{,x}'",
+    "standard:verbose": "npx standard --verbose",
+    "lint:standard": "prettier-standard --format | standard --fix",
```

Comment:
I would like to have one  script called "lint" that do everything needed to pass the test script.
Based on the above comment we could need only ts-standard task to run during the test phase, and we can also remove the lint-and-format, lint:standard, lint:s-standard and just replace all of them with one script "lint" that should look like this : 

    "lint": "ts-standard --fix  && npm run import-sort"

Reply:
Can we please do this in separate branches, otherwise it will be really chaotic here

Reply:
And for debugging purpose of each linter, having separate scripts to run them is currently useful.